deploy/pulse.conf

```diff
@@ -1,3 +1,12 @@
+# Ports:
+#
+# 3000: production app
+# 5000: production webhook
+# 6000: staging app
+# 7000: staging webhook
+#
+# 4000: https webhook
+
```

Comment:
I also updated the Pulse nginx config file to document the ports at use on the box for webhooks, and to catch the on-server change I made a couple weeks ago to fix the redirect behavior for `http://pulse.cio.gov`.